### PR TITLE
ko: Update outdated files partly in dev-1.15-ko.4

### PR DESCRIPTION
**Pull request by gochist**
Outdated files in the dev-1.15-ko.4 branch.

It fixes_ #15745 partly

#### need to be compared with `dev-1.15-ko.1`
1. [x] content/en/_index.html 1 file changed, 2 insertions(+), 2 deletions(-)
1. [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 6 insertions(+), 3 deletions(-)
1. [x] content/en/docs/concepts/overview/components.md 1 file changed, 12 insertions(+), 7 deletions(-)
1. [x] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 8 insertions(+), 8 deletions(-) - already done
1. [x] content/en/docs/concepts/overview/working-with-objects/field-selectors.md 1 file changed, 1 insertion(+), 1 deletion(-)  - already done
1. [x] content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 11 insertions(+), 4 deletions(-)
1. [x] content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 139 insertions(+), 141 deletions(-) - already done
1. [x] content/en/docs/concepts/workloads/pods/pod.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [x] content/en/docs/reference/_index.md 1 file changed, 1 insertion(+), 1 deletion(-) - already done
1. [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 7 insertions(+), 4 deletions(-)
1. [x] content/en/docs/setup/_index.md 1 file changed, 7 insertions(+), 4 deletions(-)
1. [x] content/en/docs/tasks/tools/install-minikube.md 1 file changed, 9 insertions(+), 7 deletions(-)
1. [x] content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html 1 file changed, 7 insertions(+), 7 deletions(-)
1. [x] content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html 1 file changed, 2 insertions(+), 2 deletions(-) - already done
1. [x] content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md 1 file changed, 7 insertions(+)
1. [x] content/en/examples/application/mysql/mysql-statefulset.yaml 1 file changed, 12 insertions(+), 13 deletions(-)

#### need to be compared with `dev-1.15-ko.1`
1. [x] R087 content/en/docs/concepts/configuration/scheduler-perf-tuning.md -> content/en/docs/concepts/scheduling/scheduler-perf-tuning.md

### CODE REVIEW

File: content/ko/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -158,14 +160,14 @@ Hyper-V는 다음 세 버전의 윈도우 10에서 실행할 수 있다. Windows
...
-choco install minikube kubernetes-cli
+choco install minikube
 ```
 
 Minikube 설치를 마친 후, 현재 CLI 세션을 닫고 재시작한다. Minikube 실행 파일의 경로는 실행 경로(path)에 자동으로 추가된다.
 
 ### 인스톨러 실행파일을 통한 Minikube 설치
 
-윈도우에서 수동으로 [Windows 인스톨러](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal)로 설치하려면, [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/minikube-installer.exe)를 다운로드 받고, 이 인스톨러를 실행한다.
+윈도우에서 수동으로 [Windows 인스톨러](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal)로 설치하려면, [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe)를 다운로드 받고, 이 인스톨러를 실행한다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/tools/install-minikube.md]: 일관성 유지를 위해서 Windows를 윈도우로 변경,
원문에서 누락된 내용이 있어서 추가하였습니다. 참고하시기 바랍니다.
```suggestion
[윈도우 인스톨러](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal)를 사용하여 윈도우에 Minikube를 수동으로 설치하려면, [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe)를 다운로드 받고, 이 인스톨러를 실행한다.
```

[원문]
To **install Minikube** manually on Windows using [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), download [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) and execute the installer.

**Comment by gochist** [comment]
@seokho-son 리뷰 감사드립니다. 내용 반영했습니다.

**Comment by seokho-son** [comment]
감사합니다 !

/lgtm
/approve

**Pull request merged**